### DISCUSSION TITLE
Can I change the initial value in the training neural network?

### DISCUSSION THREAD

**Discussion by ghost** [deepmodeling/deepmd-kit#1613] [Q&A]
Hi everyone,
How can I change the initial value of the neural network. We couldn't have a satisfactory result by trying almost everything, like change the parameters in the input file and add new data. So we want change this one in the neural network.

**Comment by likefallwind** [comment]
Hello, I'm not know for sure:

1. I don't think change the initial value will help. In most time, initial value only influence the training time. Maybe you can change the size of neural network.
2. If you still want to change the initial value, I guess you can write a "model.ckpt" and use this command:(https://tutorials.deepmodeling.org/en/devel/Tutorials/DeePMD-kit/learnDoc/Handson-Tutorial%28v2.0.3%29.html#train-a-model)
$ dp train  --restart model.ckpt  input.json
I haven't tried this, so I'm not sure whether this will help

**Reply by ghost** [reply]
Thank for your reply, I will have a try about this.

**Reply by ghost** [reply]
Hey, thanks for your helping. But I am thinking, if we make the number of nodes in neural network too big. Is the model possible to become overfitting?

**Reply by likefallwind** [reply]
Of course. In machine learning, you should find proper size to make sure your model is accurate but not overfitting.

**Comment by McAlexDudu** [comment]
Actually DeePMD tutorials FAQS have some test about this,

[https://docs.deepmodeling.org/projects/deepmd/en/master/troubleshooting/howtoset_netsize.html](url)

It may help.

**Reply by ghost** [reply]
Thank for your reply, I will have a try about this.
